### PR TITLE
Fix legacy apps loading issues

### PR DESCRIPTION
**Pull request by matheusps**
### Changes
- Fix legacy apps not loading on safari.
- Fix legacy apps not loading on dev workspaces.

### 🧪 Testing 
Try https://matheusv3--recorrenciaqa.myvtex.com/admin/Site/Produto.aspx on chrome & safari.

### CODE REVIEW

File: react/components/IframeLegacy.js [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+import { isSafari } from 'react-device-detect'
 
 const COMPENSATION = 42
-const getLegacyBaseURL = account => `https://${account}.vtexcommerce${getEnv() === 'beta' ? 'beta' : 'stable'}.com.br/admin/`
+const getLegacyBaseURL = (account, workspace) => {
+  const isDevWorkspace = workspace && workspace !== 'master'
+
+  const environment = isDevWorkspace ? `${workspace}--` : ''
+
+  console.log('workspace', workspace)
```
Comment (kevinch) [inline react/components/IframeLegacy.js]: Remove console statement.

File: react/components/IframeLegacy.js [hunk 2]
```diff
@@ -45,9 +67,14 @@ class IframeLegacy extends Component {
   handleIframeMessage = (event) => {
     if (event.data && event.data.type) {
       const type = event.data.type
-      console.debug(`%c [LEGACY IFRAME] \n Received iframe message with type: ${type}`, 'background: #002833; color: #bada55')
+      console.debug(
```
Comment (kevinch) [inline react/components/IframeLegacy.js]: Useful?

File: react/package.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "js-cookie": "^2.2.0",
     "prop-types": "^15.5.10",
     "ramda": "^0.25.0",
+    "react-device-detect": "^1.17.0",
```
Comment (kevinch) [inline react/package.json:13]: Did you think of `vtex.device-detector`?
https://github.com/vtex/admin-orders/blob/master/manifest.json#L14

Comment (matheusps) [reply react/package.json:13]: The vtex library does not have `isSafari, isChrome, etc...`

**Review by marcelovicentegc — APPROVED** [review]
Except for the `console.log`, LGTM! 😉

File: react/components/IframeLegacy.js (hunk 2 above)
Comment (marcelovicentegc) [reply react/components/IframeLegacy.js]: I wouldn't say so.

Comment (matheusps) [reply react/components/IframeLegacy.js]: I don't think it is

**Review by kevinch — APPROVED** [review]

**Review by lucasaarcoverde — APPROVED** [review]

**Pull request merged**